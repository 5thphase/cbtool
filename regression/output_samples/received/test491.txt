Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command ""/home/msilva/cloudbench/lib/operations//../../cbact" --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:9.2.208.21,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,password:cbpwdZ9,timeout:240,host:9.2.208.21,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,db2,SUT,A:B+_*X:Y+_*R:2,default,none,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=F2408917-F287-59B1-AAD6-263E8676CF74 --daemon". The process id is 13878.

Going to unconditionally wait for 1s (1 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (100.0%)
Waited for 1 seconds.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid

Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |94.59.149.162   |simhostd11                 |SUT           |private    |none    |none       |D02B3075-38D0-5644-A906-1E5A57B17986
|vm_3          |tinyvm              |pico32      |64.39.37.162    |simhostb1                  |SUT           |private    |none    |none       |C062BEB6-EF0C-5F3F-BEF8-3F269D5B171D
|vm_4          |db2                 |copper32    |245.121.185.43  |simhostd13                 |SUT           |private    |none    |none       |FB790174-A542-5FE5-ADA2-B73E6907A36C
|vm_5          |netclient           |platinum64  |34.8.152.115    |simhosta5                  |SUT           |private    |none    |none       |E2876805-A9D0-59A4-848F-EF95AC8CA5E2
|vm_6          |tinyvm              |pico32      |176.48.131.69   |simhosta13                 |SUT           |private    |none    |none       |BBF23E09-21F7-555D-87CB-3854F8B2E519
|vm_7          |tinyvm              |pico32      |200.137.174.143 |simhosta7                  |SUT           |private    |none    |none       |E0D00E13-CBE7-5A5F-AC8E-84EC9C74D982
|vm_8          |tinyvm              |pico32      |153.197.210.85  |simhostd1                  |SUT           |private    |none    |none       |18A25181-C4D0-57F8-A930-3806181DF4B6
|vm_9          |db2                 |platinum64  |64.228.55.53    |simhostd8                  |SUT           |private    |none    |none       |F2408917-F287-59B1-AAD6-263E8676CF74

The VM object vm_9, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhostd8                            
|meta_tags                               |{'A': 'B', 'X': 'Y', 'R': '2'}       
|role                                    |db2                                  
|size                                    |platinum64                           
|vmc_name                                |simzone_b                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |8.87M                         
|Redis Version                                        |3.0.6                         
|Uptime (in seconds)                                  |8705331                       
|Total Connections Received                           |377409                        
|Total Commands Processed                             |1223365002                    
|Number of Keys                                       |851                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |15                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |8                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |8                             
|VMs ISSUED                                           |9                             
|VMs ARRIVED                                          |9                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |5                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |218                           

 ---------------- VMC-WIDE COUNTERS ----------------
|9E74D721-5227-5C9B-A7BD-BE5B4A43D6AA (simzone_d) VM RESERVATIONS|0                             
|3B6C80F6-6318-5721-A484-E3DA77718B57 (simzone_c) VM RESERVATIONS|0                             
|803C5D90-3C59-5871-9ACF-5683E3FB4506 (simzone_a) VM RESERVATIONS|3                             
|270A5605-A7C0-5F5E-B66F-19B77C6645BE (simzone_b) VM RESERVATIONS|5                             

