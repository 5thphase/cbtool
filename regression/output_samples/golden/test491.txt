Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.1.250,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.1.250,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,db2,SUT,A:B+_*X:Y+_*R:2,default,none,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=0EA1AF57-79C7-5F87-B6D7-17F1875283DC --daemon". The process id is 26889.

Going to unconditionally wait for 1s (1 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (100.0%)
Waited for 1 seconds.
The following PENDING VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_9          |unavailable         |unavailable |unavailable     |unavailable                |unavailable         |unavailable|unavailable|0EA1AF57-79C7-5F87-B6D7-17F1875283DC

Going to unconditionally wait for 8s (8 seconds). The command line interface will be blocked during the waiting.
Waited 1 seconds... (12.5%)
Waited 2 seconds... (25.0%)
Waited 3 seconds... (37.5%)
Waited 4 seconds... (50.0%)
Waited 5 seconds... (62.5%)
Waited 6 seconds... (75.0%)
Waited 7 seconds... (87.5%)
Waited 8 seconds... (100.0%)
Waited for 8 seconds.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |180.159.11.28   |simhosta1                  |SUT                 |none    |none       |5C29CE0D-B174-5AD8-BF75-6B1793833BBF
|vm_3          |tinyvm              |pico32      |84.243.161.209  |simhostb1                  |SUT                 |none    |none       |EE9F2972-129F-5020-A9CA-5FFD376C69A9
|vm_4          |db2                 |copper32    |25.107.28.157   |simhostb1                  |SUT                 |none    |none       |FDB6A9EA-8A7A-5DAB-ADAA-CE2AECFF01DA
|vm_5          |netclient           |platinum64  |121.217.11.74   |simhostd3                  |SUT                 |none    |none       |1E7D7D70-5AC8-555B-A53B-0C831E5522E8
|vm_6          |tinyvm              |pico32      |7.228.165.122   |simhostc3                  |SUT                 |none    |none       |D50569B7-E411-5214-83AF-E6653495906C
|vm_7          |tinyvm              |pico32      |70.230.20.236   |simhosta3                  |SUT                 |none    |none       |318B9CD3-277A-5943-B19E-71F871D639E7
|vm_8          |tinyvm              |pico32      |247.34.81.128   |simhosta2                  |SUT                 |none    |none       |C345ACE0-B563-56B5-A4F5-4A6F3F49858C
|vm_9          |db2                 |platinum64  |143.13.76.80    |simhostd2                  |SUT                 |none    |none       |0EA1AF57-79C7-5F87-B6D7-17F1875283DC

The VM object vm_9, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhostd2                            
|meta_tags                               |{'A': 'B', 'X': 'Y', 'R': '2'}       
|role                                    |db2                                  
|size                                    |platinum64                           
|vmc_name                                |simzone_c                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |2.05M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |3793109                       
|Total Connections Received                           |81298                         
|Total Commands Processed                             |6025421                       
|Number of Keys                                       |357                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |2158592                       
|Data Size                                            |500256                        
|Index Size                                           |425152                        
|Average Object Size                                  |507.359026369                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |8                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |8                             
|VMs ARRIVED                                          |9                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |98                            

 ---------------- VMC-WIDE COUNTERS ----------------
|516A5A15-B093-5B5F-B171-EE9AA82AAEF0 (simzone_c) VM RESERVATIONS|3                             
|8D79A9F1-34B8-5D1A-A903-082245CE052C (simzone_d) VM RESERVATIONS|0                             
|B0B6FF0A-FAE8-5217-A335-C0F1B0BEA0D6 (simzone_b) VM RESERVATIONS|4                             
|D5506DC9-493A-57EE-86F6-0B96CB07D162 (simzone_a) VM RESERVATIONS|1                             

